README.md

```diff
@@ -180,7 +181,7 @@ end
 
 Then to use the component in your code, add the following line to your `.h` or `.m` files:
 
-```
+```swift
```

Comment:
thanks for noticing the missing language specifier. In this case, this code is Objective C. Could you please change this to `objc` over `swift`?